<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositCalc</class>
 <widget class="QDialog" name="depositCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E28DC2, stop: 1 #D9ABC7);
}</string>
  </property>
  <widget class="QRadioButton" name="is_capitalized">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Капитализация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accured_interest">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
   border: 1px solid pink;
   border-radius: 10px;
   background-color: rgb(255, 239, 213);
   qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
   border: 1px solid pink;
   border-radius: 10px;
   background-color: rgb(255, 239, 213);
   qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_amount">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
   border: 1px solid pink;
   border-radius: 10px;
   background-color: rgb(255, 239, 213);
   qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вклад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процент</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Периодичность</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Снятия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Налог(%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Прибыль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Налог</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Итоговая сумма</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   border: 1px solid pink;
   border-radius: 10px;
   background-color: rgb(255, 239, 213);
}</string>
   </property>
   <property name="text">
    <string>Расчёт</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="initial_deposit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="period_in_months">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="initial_interest">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="payment_periodicity">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="replenishment_amount">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="withdraw_amount">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tax_interest">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
